lib/components/SelectionControl/checkbox/checkbox-single/index.tsx

```diff
@@ -89,8 +89,6 @@ export const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
 
         const onKeyDown: React.KeyboardEventHandler<HTMLInputElement> = (e) => {
             e.stopPropagation();
-            e.preventDefault();
```

Comment:
Check the original reason why we needed to add this in the first place.

Reply:
There is a well-known behavior, that `spacebar` (from keyboard) is triggering `click` event on such elements like input ([Stackoverflow](https://stackoverflow.com/questions/27878940/spacebar-triggering-click-event-on-checkbox) topic, [article](https://www.w3.org/TR/WCAG20-TECHS/SCR35)). That was made for accessibility purposes: browsers use the click event even if its triggered from the keyboard in the event where a mouse isn't present. 
Because of that, when user hit `Space`, `onChange` was triggered twice (as a `keyboard` and as a `mouse` event). I didn't want this behavior and add `e.preventDefault();` inside of `onKeyDown` in order not to trigger `onChange` twice. That was the original reason.
But it turned out, that `e.preventDefault();` also disabled `focus` from jumping to another element when user is pressing 'Tab', so after all I decided to remove `e.preventDefault();` and to remove `e.key === KEY.SPACE` from check, because of the reasons mentioned above (hitting space will trigger `onChange` anyway)

Reply:
Can we detect if the even came from the spacebar and that's the only time we will fire `e.preventDefault();`

Reply:
Yep, good catch, thanks! It should work, let me add

Reply:
Sorry for my stupidness)

Reply:
You did a great job checking the UI events, thanks for that!